docker/appserver/Tomcat/src/scripts/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+if [ "$1" = 'catalina.sh' ]; then
+	# Fix permissions
+	if [[ "${SET_PERMISSIONS_ON_STARTUP}" != 'FALSE' ]]; then
+		echo Setting permissions on startup
+		echo To improve startup time, run /setPermissions.sh and set environment variable SET_PERMISSIONS_ON_STARTUP to FALSE as last step in the Dockerfile to skip this step 
+		/setPermissions.sh
+	fi
+	
+	exec gosu tomcat "$@"
+fi
```

Comment:
Waarom start je alleen catalina als tomcat en al het eventuele andere niet?
Misschien goed om dat er ook in een commentaar bij te zetten.

Reply:
Commentaar is toegevoegd, ook in de Dockerfile staat nu een regel over het hoe en waarom van de entrypoint.sh